modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -101,41 +121,57 @@ image::sso-configuration-tutorials/oidc-okta-successful-login.png[title="Okta OI
 
 === ID token
 
-This example shows how to configure Okta for authentication and authorization using ID tokens.
+This example shows how to configure Okta for authentication and authorization using ID tokens and the how to configure Neo4j to use them.
 
-. Follow the first two steps from the instructions for xref:#_access_token[Access token].
+==== Add a groups claim to ID tokens
 
-. Create the claims as indicated:
-+
-image::sso-configuration-tutorials/okta-claims.svg[title="Okta claim creation panel"]
+You can add a groups claim to ID tokens to configure authentication and authorization using ID tokens.
+
+. From the left-hand side of the Okta dashboard, navigate to *Security -> API*.
+. Click the default authorization server (the one that shows `api://default` as audience) to return the `groups` claim in access tokens:
+.. On the *Claims* tab, click *Add Claim*.
+.. Add a claim with the name `groups`.
+.. From the *Include in token type* dropdown, select *ID Token*.
+.. From the *Value type* dropdown, select *Groups*.
+.. From the Filter dropdown, select *Matches regex* and the value `.*`.
+.. Click *Create*.
+. Add a claim with the name `userid` and the value type `User ID`.
 +
 [NOTE]
 ====
-In the case of access tokens, a default sub is already provided automatically.
-However, for ID tokens, the name you give to your claim needs to be also indicated in the configuration `dbms.security.oidc.okta.claims.username=userid`.
+The `userid` claim is not included in the ID token by default like the default `sub` claim for access tokens, thus you need to add it manually.
+The name you give to your claim needs to be also indicated in the configuration `dbms.security.oidc.okta.claims.username=userid`.
```

Comment:
maybe specify that it is the neo4j configuration file. `_neo4j.conf_ file:` as we do above.